reflex/app.py

```diff
@@ -633,10 +649,14 @@ def add_custom_404_page(
             on_load: The event handler(s) that will be called each time the page load.
             meta: The metadata of the page.
         """
-        if component is None:
-            component = Default404Page.create()
+        console.deprecate(
+            feature_name="App.add_custom_404_page",
+            reason=f"Use app.add_page(component, route=/{constants.Page404.SLUG!r}) instead.",
```

Comment:
this puts the slash outside the quotes

```
"Use app.add_page(component, route=/'404') instead."
```

either leave the leading slash off (it works just the same), or do the quoting yourself (without the `!r`)

```suggestion
            reason=f"Use app.add_page(component, route='/{constants.Page404.SLUG}') instead.",
```